examples/web-wagmi/src/utils/media.ts

```diff
@@ -0,0 +1,65 @@
+import { ContentPublication, Overwrite } from '@lens-protocol/react-web';
+
+import { resolveFullResourceUrl } from './resolveFullResourceUrl';
+
+export const isInEnum = <TEnumValue extends string | number>(
+  enumVariables: { [key: string]: TEnumValue },
+  value: string | number,
+): value is TEnumValue => {
+  return Object.values<string | number>(enumVariables).includes(value);
+};
+
+export enum ImageType {
+  JPEG = 'image/jpeg',
+  JPG = 'image/jpg',
+  PNG = 'image/png',
+  WEBP = 'image/webp',
+  GIF = 'image/gif',
+}
+
+export enum VideoType {
+  MP4 = 'video/mp4',
+}
+
+export enum AudioType {
+  WAV = 'audio/wav',
+  MP3 = 'audio/mpeg',
+  OGG = 'audio/ogg',
+}
```

Comment:
These are exported by the `@lens-protocol/react` and `@lens-protocol/react-web` packages. Any reason to repeat these?

Reply:
fixed